test/conftest.py

```diff
@@ -1,17 +1,15 @@
-import pytest
-import os
-import shutil
-import tempfile
+from data import CACHED_SERVICES
 from time import time
 import warnings
+import tempfile
 import logging
-
+import pytest
+import shutil
 import quest
+import os
 
-from data import CACHED_SERVICES
 
-base_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
@AaronV77 The tests are failing because you've removed these lines. These are providing the path to the `./test/files` directory where the test data is stored.